react/features/toolbox/components/VideoMuteButton.js

```diff
@@ -164,10 +164,14 @@ class VideoMuteButton extends AbstractVideoMuteButton<Props, *> {
 function _mapStateToProps(state): Object {
     const { enabled: audioOnly } = state['features/base/audio-only'];
     const tracks = state['features/base/tracks'];
+    const videoTrack = getLocalVideoTrack(tracks);
+    const mediaType = videoTrack && videoTrack.videoType === 'desktop'
+        ? MEDIA_TYPE.PRESENTER
+        : MEDIA_TYPE.VIDEO;
 
     return {
         _audioOnly: Boolean(audioOnly),
-        _videoMuted: isLocalTrackMuted(tracks, MEDIA_TYPE.VIDEO)
+        _videoMuted: isLocalTrackMuted(tracks, mediaType)
```

Comment:
WDYT about adding some `isLocalVideoMuted(tracks)` which does this media type dance?

Reply:
Done, I added isLocalVideoTrackMuted(tracks) method to check for mute status on the presenter track if present, otherwise it returns the mute status of the video track.